sjsonnet/src-jvm-native/sjsonnet/CachedResolvedFile.scala

```diff
@@ -61,12 +61,12 @@ class CachedResolvedFile(val resolvedImportPath: OsPath, memoryLimitBytes: Long,
     }
   }
 
-  override lazy val contentHash: String = {
```

Comment:
we loose the caching here. Any way to restore the lazy val logic?

Reply:
I had to modify the parent trait to use a parens-less function.